vaadin-combo-box-flow-demo/src/main/java/com/vaadin/flow/component/combobox/demo/ComboBoxView.java

```diff
@@ -226,10 +227,7 @@ private void storingCustomValues() {
         // begin-source-example
         // source-example-heading: Storing custom values
         ComboBox<Project> comboBox = new ComboBox<>("Project");
-        DataProvider<Project, String> dataProvider = DataProvider
-                .fromFilteringCallbacks(this::fetchProjects,
-                        this::countProjects);
-        comboBox.setDataProvider(dataProvider);
+        comboBox.setItems(this::fetchProjects, this::countProjects);
```

Comment:
Just as a note - I think the CB demos should be similar to grid - show the usage without the count query and then show how to get to explicit count. And maybe even the estimate API.

Reply:
Yes, all those mentioned demos added.